### ISSUE TITLE
Manual Transparency Report for 2021 Annual Report

### ISSUE THREAD

**Issue by sallytay** [mysociety/whatdotheyknow-theme#910]
For this year's mySociety Annual Report we are planning to put together a manual Transparency Report.

We are doing a manual version at this stage this because it will allow use to create something within the deadline for the annual report and will also help to shape the automated version that we aspire to  and would like to implement in the future, as detailed in https://github.com/mysociety/alaveteli/issues/2658

The scope for this, will be much smaller than a full automated version but we hope will demonstrate Transparency in how we operate WDTK. With this in mind the scope will be:

* ~3 high level stats + ~5 short paragraphs of text, based around some stats.

It will also need to be based around the information that we can currently get from the system in time period that we have.

I'm working on a proposal of what these could be, which I will share early next week. 

For reference the deadlines are:
The annual report is scheduled to go out on 16 December
Design is scheduled to be completed by 9 December
Ideally this means that copy should be ready by 2 December

Sally

**Comment by RichardTaylor** [comment]
How about a couple of brief case studies?  The following are fictional (inspired by real cases, and there may not be relevant cases in the year in question) but the kind of thing we could publish:

>A council released a copy of its landlord licence register, we removed a name and address from the material released after an individual told us they had been a police officer who'd dealt with serious crime and they feared those they'd dealt with in their professional career finding their address and causing them harm. 

>Ten landlords contacted us claiming our publication of their addresses in landlord licence holder databases released by councils under FOI were causing them a nuisance as they thought some junk mail they were receiving was due to the information we were publishing. We continued to publish the information, considering the impact on the individuals did not outweigh our interests, and the public interest, in continued publication. 

>A council released information relating to a restaurant environmental heath inspection. The restaurateur claimed the material released was defamatory. We were not prepared to take the legal risk involved in continued publication of the material but made clear on the thread where it had been removed that we are happy to make it available to responsible journalists, and elected representatives, on request.   

>Five people requested we remove their names from university graduation programmes in which their degree subject, and degree class were listed. We carefully assessed each request on its merits and decided to continue publication in four of the five cases, removing a name from the released document in the other instance.  

Other information we could include:

* How many complaints have been made to the ICO, and how many have been upheld
* How many times have we had action against us started in court.  How many court cases have we won / lost.  (Hopefully this will be zero, but it's an important zero I think). 

I'd like to see the total number of GDPR personal information take-down requests noted - perhaps both the total per year, and present at as the average per working day too? 

Similarly the total number of defamation cases dealt with in the year could be noted.  

We could quote the number of police / law enforcement requests for user-data and the outcomes. Key context would be if they related to legitimate use of our service, or were tangential to it.  

We might also want to consider what we can say about any requests for information we are not permitted to talk about.

**Comment by RichardTaylor** [comment]
Suggestion - include a statement explaining how we strive to run the site as transparently as possible, following a principle of minimal redaction, and where possible show on request threads where we have removed material, and why.

**Comment by sallytay** [comment]
The content of the proposed report has now been put together.  

This is not how the final version will look, this just details of the content

We may not be able to include everything this year due to the deadlines, resources and ability to retrieve the data that we need. However we will be focusing on using this as the groundwork for the final goal of the automated reported, and items that we may not be able to include this year will hopefully be included in subsequent years.

Here is the document https://docs.google.com/document/d/1cfTia3YIR_Sv0uunAnd5ZtSS8z04Fo3LlHxMNMcITG4/edit. This should be open if you are logged in as the WDTK Support Inbox or a mySociety user.

Comments and further suggestions are welcomed. Bearing in mind the content limit for this first ~3 high level stats + ~5 short paragraphs of text, based around some stats.

Gareth and I will be working on getting the date and writing up any text that is needed.

Sally

**Comment by RichardTaylor** [comment]
The three high level figures proposed are:

1. Total number of requests taken down from WDTK in 2021 (or % of total, probably need to add total for context?) (Ticket #922)
2. Total number of requests ‘censored’ in 2021 (due to GDPR, Defamation, etc)  (Ticket #922)
3. Total number of users banned for site misuse (maybe need total users for context) (Ticket #925)

Alternative proposal for the three high-level figures:

1. Number, and fraction, of correspondence threads either removed or redacted
2. Number of GDPR takedown requests
3. Number of defamation takedown requests rejected

Another alternative:

1. Number, and fraction, of correspondence threads either removed or redacted 
2. Number of takedown requests
3. Fraction of takedown requests rejected  « I suspect we might not be able to calculate this easily

Warnings on doing calculations based on the stats eg. noting takedown requests may relate to many correspondence threads might be appropriate. 

Is spam significant enough to exclude from takedown / redaction stats.

**Comment by RichardTaylor** [comment]
Linking https://github.com/mysociety/whatdotheyknow-theme/issues/909

**Comment by sallytay** [comment]
I've completed the draft of the Transparency report with the data available.

I've not changed the 3 high level stats for this first attempt as I don't think we have reliable and easy to obtain data for these these - however these should definitely be considered moving forward.

I plan to talk though this with @garethrees to ensure I've interpreted and explained the data properly, but as always any comments and suggestions welcomed.

xhttps://docs.google.com/document/d/1cfTia3YIR_Sv0uunAnd5ZtSS8z04Fo3LlHxMNMcITG4/edit

Sally
